pkg/clicmd/root.go

```diff
@@ -106,7 +117,8 @@ func NewRoot(appFs afero.Fs, wd string, args []string) (*cobra.Command, error) {
 		return nil, err
 	}
 
-	if len(args) > 0 && cmdName != "init" {
+	cmds := []string{"init", "version"}
+	if len(args) > 0 && !(strings.InSlice(cmdName, cmds) || containsHelp(args)) {
```

Comment:
This is getting hard to read.
It's a matter of taste, but my preference would be something like this:

```go
switch {
case string.InSlice(cmdName, cmds):
    // commands that should not load an app
case containsHelp(args):
    // help does not require an app
case cmdName != "":
    a, err = app.Load(appFs, wd, false)
    ...
default:
    // noop
}
```